aimmo-game-creator/worker_manager.py

```diff
@@ -168,10 +169,18 @@ class KubernetesWorkerManager(WorkerManager):
 
     def __init__(self, *args, **kwargs):
         self._api = pykube.HTTPClient(pykube.KubeConfig.from_service_account())
+        kubernetes.config.load_incluster_config()
+        self._api_instance = kubernetes.client.ExtensionsV1beta1Api()
```

Comment:
food for thought - is is stable? looks like they might rename that methid anytime - should we bind all the BetaThing named methods with a more generic name so it's easier to upgrade later?

Reply:
saw v1beta1 it is written in the ingress YAML file, but kubernetes is installed as latest version in the game creator. But Fixed version in minikube requirement...?

Reply:
Discussed earlier today. Wrapper adds complexity to debugging and that's the official API name.